crowsnest/components/cam.py

```
@@ -0,0 +1,71 @@
+            await lock.acquire()
+            self.log_quiet(
+                f"Start {self.streamer.keyword} with device "
+                f"{self.streamer.parameters['device']} ..."
+            )
+            watchdog.configured_devices.append(self.streamer.parameters["device"])
+            process = await self.streamer.execute(lock)
+            if process:
+                await process.wait()
+            else:
+                self.log_error(f"Start of {self.parameters['mode']} failed!")
+        except Exception:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)